src/metabase/models/persisted_info.clj

```diff
@@ -74,14 +75,28 @@
        (take 10)
        (apply str)))
 
+;; Without the :cache-granular-controls feature flag, any models with state="off" should be treated as if they were
+;; in state="creating" (i.e. they should be refreshable, and not prunable).
```

Comment:
this comment look like it could be under `refreshable-states` docstring.

Also, I'd be more verbose about why `off` state could exist in OSS instances.